portfolio.js

```diff
@@ -23,3 +23,28 @@ projectInfo.forEach(function(currentProject) {
 projects.forEach(function(project) {
   $('#projects').append(project.toHtml());
 });
+
+Project.loadAll() = function(projects) {
+  projects.forEach(function(ele) {
+    projects.push(new Project(ele));
+  });
+};
```

Comment:
You currently have a forEach call sitting above (line 19) this method definition that is essentially doing the same thing. You should get rid of that statement and keep that functionality inside of Project.loadAll.